docs/packages/pylhcsubmitter/autosix.md

```diff
@@ -0,0 +1,115 @@
+    madx = Madx()
+    # setup your study
+    madx.globals["VRF400"] = 8 if madx.globals["NRJ"] < 5000 else 16
+    madx.globals["LAGRF400.B1"] = 0.5
+    madx.globals["LAGRF400.B2"] = 0.
+    madx.twiss()  # used by sixtrack
+    madx.command.sixtrack(cavall=True, radius=0.017)
+    ```
+
+## What Happens After Submitting
+
+Upon running the script, the **Jobs.tfs** file is created similarly to `job_submitter`, and the following stages are run per job:
+
+1. `create_jobs`: create workspace, fill sysenv, sixdeskenv and mask.
+2. `initialize_workspace`: initialize workspace from the env-files.
+3. `submit_mask`: submit job to `HTCondor` from filled mask. (*interrupt*)
+4. `check_input`: check if sixdesk input is complete.
+5. `submit_sixtrack`: submit sixdesk jobs to `HTCondor`. (*interrupt*)
+6. `check_sixtrack_output`: check if all sixdesk jobs are completed.
+7. `sixdb_load`: create database and load jobs output.
+8. `sixdb_cmd`: calculate DA from database data.
+9. `post_process`: extract data from database, write into **.tfs** files and plot.
+10. `final`: announce every stage has completed.
+
+To keep track of the stages, they are written into a **stages\_completed.txt** file in the `autosix\_output` directory in the workspaces.
+Stages that are written in this file are assumed to be done and will be skipped.
+To rerun a stage, delete its entry and that of all following stages in the **stages\_completed.txt** file and start your run anew.
+
+??? info "Execution Order"
+    The stages are ran independently of each job, meaning different jobs can be at different stages.
+    For instance, if one job has all data for the `six_db` analysis already, but the others are still running on `Sixdesk`, the `check_sixtrack_output` stage will fail for these jobs but continue for the job that is ready.
+
+Because the stages after `submit_mask` and `submit_sixtrack` need only be ran after the jobs on `HTCondor` are completed, they will interrupt execution if previous stages have successfully finished.
+To have `AutoSix` continue its work, check your scheduler via `condor_q` and run your script again after everything is done.
+It will pick up where it left as written in the **stages\_completed.txt** file.
+
+!!! tip
+    While most studies should be fine with the input options given, there is the possibility to manually adapt the **sixdeskenv** and **sysenv** files before workspace initialization by using the `stop_workspace_init` flag.
+    This will not reset automatically and one will have to remove the switch again to continue.
+    As this is a bit of a workaround, it might be easier to define a new variable in the **pylhc_submitter.sixdesk_tools.mask_sixdeskenv** file and put its current default into **pylhc_submitter.constants.autosix.SIXENV_DEFAULT**.
+    One can then use the `replace_dict` to change its value.
+    This is left open as a task for the inspired user to implement, as only he knows which variable he or she needs to change.
```

Comment:
I have left this segment untouched because I'm not sure I get what this is trying to tell me. @JoschD

Reply:
You have to have been there to understand...